harness/determined/cli/user.py

```diff
@@ -158,7 +158,15 @@ def create_user(args: argparse.Namespace) -> None:
     username = args.username
     admin = bool(args.admin)
     remote = bool(args.remote)
-    d.create_user(username=username, admin=admin, remote=remote)
+    password = args.password
+
+    if not remote and not password:
+        password = getpass.getpass("New password for user '{}': ".format(username))
```

Comment:
Nit: When I see the word "new", it makes me think do I have an old password? Since this is creating a user, we may not need to include "new":
```suggestion
        password = getpass.getpass("Password for user '{}': ".format(username))
```